src/prefect/workers/server.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import FastAPI
+import uvicorn
+
+app = FastAPI()
+
+
+@app.get("/health")
+async def check_health():
+    return {"status": "OK"}
```

Comment:
Ideally this would have access to the worker and check the last time the worker polled successfully